python/upload_with_authentication.py

```diff
@@ -72,16 +72,16 @@
     except KeyError:
         print("You are missing one of the environment variables MAPILLARY_USERNAME, MAPILLARY_PERMISSION_HASH or MAPILLARY_SIGNATURE_HASH. These are required.")
         sys.exit()
-
-    if os.path.isfile('SEQUENCE'):
+    sequence_file=os.path.join(path,'SEQUENCE')
+    if os.path.isfile(sequence_file):
         # read previous sequence UUID from the file
         print('--- SEQUENCE FILE FOUND, RESUMING UPLOAD ---')
-        with file('SEQUENCE') as f:
+        with file(sequence_file) as f:
```

Comment:
I will merge this after this minor tweaking from you. Thanks for fixing this and sorry for the late response.